requirements/edx/github.txt

```diff
@@ -50,7 +50,7 @@
 
 # Third-party:
 git+https://github.com/jazzband/django-pipeline.git@d068a019169c9de5ee20ece041a6dea236422852#egg=django-pipeline==1.5.3
--e git+https://github.com/edx/django-wiki.git@v0.0.16#egg=django-wiki
+-e git+https://github.com/edx/django-wiki.git@913c386e94dac690f471eec36b1afde1824bdea2#egg=django-wiki
```

Comment:
I assume this will be changing to `0.0.17` before merging?

Reply:
Correct, just tagged that and updating this PR now.